### PR TITLE
little cleanups

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: src/generate/register.rs [hunk 1]
```diff
@@ -158,21 +156,21 @@ fn api_docs(
...
             doc,
-            "You can {} this register and get [`{module}::R`]{}. ",
+            "You can {} this register and get [`{module}::R`]{}.",
             method("read"),
             if inmodule { "(R)" } else { "" },
         )?;
 
         if let Some(action) = read_action {
-            doc.push_str("WARN: ");
+            doc.push_str(" WARN: ");
```
Comment (Emilgardis) [inline src/generate/register.rs:173]: We could make this a little better with https://github.com/rust-lang/rust/pull/106561 now :D

Comment (burrbull) [reply src/generate/register.rs:173]: It's interesting. I'll test. The question is should we up MSRV or not?

Comment (Emilgardis) [reply src/generate/register.rs:173]: since it's just css it doesn't break when the class doesn't exist. No msrv bump needed

Comment (burrbull) [reply src/generate/register.rs]: Seems like done.
Do you have any other ideas on improving documentation?

**Review by Emilgardis — APPROVED** [review]

**Pull request merged**